ibis/common/tests/test_patterns.py

```diff
@@ -97,6 +104,30 @@ def __eq__(self, other):
         return self.__class__ == other.__class__ and self.min == other.min
 
 
+x = Variable("x")
+y = Variable("y")
+z = Variable("z")
+
+
+def test_always():
+    p = Always()
+    assert p.match(1, context={}) == 1
+    assert p.match(2, context={}) == 2
+
+
+def test_never():
+    p = Never()
+    assert p.match(1, context={}) is NoMatch
+    assert p.match(2, context={}) is NoMatch
+
+
+def test_just():
+    p = Just(1)
```

Comment:
How about `Const`?

Reply:
Or `Exactly`? I'm not settled with the naming, we should make these more descriptive. Though in this particular case `Just` is implicitly defined in most of the cases.

Reply:
This is fine!